### ISSUE TITLE
Add aplasia/hypoplasia of the calvaria

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10530]
**Preferred term label:**
Aplasia/hypoplasia of the calvaria

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
Absence or underdevelopment of the calvaria. 

**Parent term (use hpo.jax.org/app)**
Aplasia/Hypoplasia involving bones of the skull HP:0009116

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cleidocranial dysplasia (omim119600), PMID16463420

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl
Adding new term: 
- ID: HP:6000872
- label: Aplasia/Hypoplasia of the calvaria
- parents: [Abnormal calvaria morphology (HP:0002683)](https://hpo.jax.org/app/browse/term/HP:0002683) | [Aplasia/Hypoplasia involving bones of the skull (HP:0009116)](https://hpo.jax.org/app/browse/term/HP:0009116)
- pmid: 
- def: A developmental defect characterized by absence or underdevelopment of the calvaria.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10530
- synonyms:
